### PR TITLE
feat(autoware_adapi_v1_msgs): add MRM pull over behavior for mrm state

### PR DESCRIPTION
**Pull request by asana17**
## Description
Add MRM pull over behavior to MrmState.msg. 

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by isamu-takagi** [comment]
@asana17 Could you please also update the documentation?
https://github.com/autowarefoundation/autoware-documentation/blob/main/docs/design/autoware-interfaces/ad-api/features/fail-safe.md#behavior

File: autoware_adapi_v1_msgs/system/msg/MrmState.msg [hunk 1]
```diff
@@ -13,6 +13,7 @@ uint16 MRM_FAILED = 4
 uint16 NONE = 1
 uint16 EMERGENCY_STOP = 2
 uint16 COMFORTABLE_STOP = 3
+uint16 PULL_OVER = 3
```
Comment (HansRobo) [inline autoware_adapi_v1_msgs/system/msg/MrmState.msg:16]: @asana17 
`COMFORTABLE_STOP` and `PULL_OVER` have the same value.
Is this correct?

Comment (HansRobo) [reply autoware_adapi_v1_msgs/system/msg/MrmState.msg:16]:
```suggestion
uint16 PULL_OVER = 4
```

Comment (asana17) [reply autoware_adapi_v1_msgs/system/msg/MrmState.msg]: I made the mistake. Thank you for the fix.

**Comment by asana17** [comment]
> @asana17 Could you please also update the documentation? https://github.com/autowarefoundation/autoware-documentation/blob/main/docs/design/autoware-interfaces/ad-api/features/fail-safe.md#behavior

Thank you for the reply.
I added description to the documentation.
https://github.com/autowarefoundation/autoware-documentation/pull/461

**Review by isamu-takagi — APPROVED** [review]

**Pull request merged**